Sources/ComposableArchitecture/Store.swift

```diff
@@ -362,15 +362,30 @@ public final class Store<State, Action> {
     self.scope(state: toLocalState, action: { $0 })
   }
 
-  func send(_ action: Action, originatingFrom originatingAction: Action? = nil) {
+  func send(_ action: Action, originatingFrom originatingAction: Action? = nil, instrumentation: Instrumentation = .shared) {
     self.threadCheck(status: .send(action, originatingAction: originatingAction))
 
+    let eventInfo = Instrumentation.EventInfo(
+      type: String(describing: Store.self),
+      action: debugCaseOutput(action),
+      tags: originatingAction.map { ["originating": debugCaseOutput($0)] } ?? [:]
```

Comment:
since tags is only used for originatingAction IMO make that part of the Instrumentation API (so that it can be free to pass it to a noop instrumentation)

Reply:
My move to using "context" objects actually lost this and thus we can remove the `tags` stuff entirely.

However, I do like your idea of making the `originatingAction` a part of the API in general! Will update to include that. Most of the time will be a `nil` value but that is fine.